Vagrantfile

```
@@ -38,11 +47,11 @@ Vagrant.configure(2) do |config|
 
   # Install Citadel
   config.vm.provision "shell", inline: <<-SHELL
-    apt-get install -y fswatch rsync jq
+    apt-get install -y fswatch rsync jq python3-dacite
     cd /vagrant/runcitadel/core
     sudo NETWORK=regtest ./scripts/configure
     docker-compose build --parallel
```

```suggestion
    docker compose build --parallel
```

We don't use docker-compose anymore, but the newer `docker compose` plugin to docker cli, because it is v2 of `docker-compose`. This should fix the issue with `UNSAFE_REMOVE_CORS_CHECK`